example/.wordpress-network

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env hocker
+# vim:set ft=sh:
+
+hocker_network_create wordpress \
+	--driver overlay \
+	--attachable
```

Comment:
I was on the fence as to whether this should be `overlay` or `bridge` -- any preference?  I guess hocker is currently fairly focused on the single-node story, so perhaps `bridge` makes a better example?